ansible/configs/ansible-multitier-infra/configure_tower.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: Include Tower Setup Roles
+  include_role:
+    name: "{{ __tower_setup_roles }}"
+  loop:
+    - tower-license-injector
+    - tower-user-create
+    - tower-project-create
+    - tower-inventory-create
+  loop_control:
+    loop_var: __tower_setup_roles
```

Comment:
Why oh why do we need to start vars out with multiple _?? Is this one? Is it 3? What is the purpose? It really makes things harder to read at a glance and easier to typo. I strongly suggest changing those.

Reply:
This is is a widespread pattern that gives a degree of data hiding and protection. I notice  a classic role I use such as Jeff Geerling's Postgres uses __ with over 1/2 million downloads. IMHO it is a *good practice* and we discuss it whilst teaching. During Tech Ex or Summit, we saw issues because some internal role vars were *not* hidden.

Reply:
@tonykay I assume we're talking about https://galaxy.ansible.com/geerlingguy/postgresql? 

Well, it is not a popularity contest so I don't really care how many downloads it has. It also only has a community score of 3.7, so clearly room for improvement if we are looking at these kind of metrics. None of the vars that he documents in the readme that people would change use multiple underscores. They are all easily human readable and obvious.

If you guys want to use _______ for some var that you never expect people to change that is fine. If it is for a var that people would be expected or _enabled_ to change (like instance type), I am firmly against naming vars this way. It is just NOT intuitive or easy to read. It also serves no purpose for "data hiding and protection". Why would you want to hide something like that?

If this is for a specific Ansible course/lab, I can release it. But I would prefer to not merge this for a general use config without further discussion on variable naming standards like this.

Reply:
also, in the contribution guidelines we updated a few weeks ago, we made a point in discussing variable naming standards. The agreement was that vars should be prefixed with the role or config name. We've been looser enforcing with config vars and stricter with role vars. But we really should be doing it in both consistently.

Reply:
These are the emerging standards we are trying to align with:

https://github.com/oasis-roles/meta_standards

The key thing is that the underscore indicates that this is a "private" variable. It only exists for the scope of this one task and goes away after it. The standards to prefix variable names with role applies when the variable is part of an interface and is exposed outside of a task context.

Reply:
That is for vars isn't it, this is a private internal var, not a role var. Maybe we should take this out of that github comments. I'll ping you and the usual suspects.

Reply:
Ok, so for internal vars that will never be exposed to anything ok since that is a standard. Lets address the exposed vars in the other comments.